### PR TITLE
adding TC  for llama-2-13b-chat

### PR DESCRIPTION
**Pull request by mwaykole**

### CODE REVIEW

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_models.robot [hunk 1]
```diff
@@ -269,6 +269,40 @@ Verify User Can Serve And Query A codellama/codellama-34b-instruct-hf Model
...
+    ...    limits_dict=${limits}    kserve_mode=${KSERVE_MODE}
+    Deploy Model Via CLI    isvc_filepath=${INFERENCESERVICE_FILLED_FILEPATH}
+    ...    namespace=${test_namespace}
+    Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${model_name}
+    ...    namespace=${test_namespace}    timeout=900s
+    Run Keyword If    "${KSERVE_MODE}"=="RawDeployment"
+    ...    Start Port-forwarding    namespace=${test_namespace}    model_name=${model_name}
+    Query Model Multiple Times    model_name=${model_name}    runtime=${TGIS_RUNTIME_NAME}
+    ...    inference_type=all-tokens    n_times=1    protocol=grpc
+    ...    namespace=${test_namespace}   query_idx=4    validate_response=${TRUE}    # temp
```
[the remark this answers is not in the document]
Comment (mwaykole) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_models.robot:294]: yess i need to change it i was waiting for the cluster

[the remark this answers is not in the document]
Comment (mwaykole) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_models.robot]: done

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_models.robot [hunk 2]
```diff
@@ -269,6 +269,40 @@ Verify User Can Serve And Query A codellama/codellama-34b-instruct-hf Model
     ...    AND
     ...    Run Keyword If    "${KSERVE_MODE}"=="RawDeployment"    Terminate Process    llm-query-process    kill=true
 
+Verify User Can Serve And Query A meta-llama/llama-2-13b-chat Model
+    [Documentation]    Basic tests for preparing, deploying and querying a LLM model
+    ...                using Kserve and TGIS standalone runtime
+    [Tags]    Tier1    RHOAIENG-3483
```
Comment (tarukumar) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_models.robot:275]: Remove tier1 tag

**Comment by mwaykole** [comment]
tested with 
rhods-ci-pr-test/2586

**Review by bdattoma — APPROVED** [review]

**Review by tarukumar — APPROVED** [review]

**Review by bdattoma — APPROVED** [review]

**Pull request merged**